### ISSUE TITLE
Lua 'Unknown Phase' Error from tikzfeynman.patch.lua

### ISSUE THREAD

**Issue by JP-Ellis** [JP-Ellis/tikz-feynman#12]
Reported initially by @cm-cc in #11.

Compiling with LuaLaTeX on Ubuntu 15.10 results in an error.  The log is [here](https://gist.github.com/cm-cc/a5afb86ff53c2b0b9ce7) and the error in particular is:

```log
! LuaTeX error ...carlo/texmf/tex/latex/tikz-feynman/tikzfeynman.patch.lua:966: 
unknown phase
stack traceback:
	[C]: in function 'assert'
	...carlo/texmf/tex/latex/tikz-feynman/tikzfeynman.patch.lua:966: in function 'h
andle'
	...carlo/texmf/tex/latex/tikz-feynman/tikzfeynman.patch.lua:979: in function 'g
et_current_options_table'
	...carlo/texmf/tex/latex/tikz-feynman/tikzfeynman.patch.lua:139: in function 'b
eginGraphDrawingScope'
	[\directlua]:1: in main chunk.
\pgfgdbeginscope ...@gd@parameter@stack@height ) }
                                                  \begingroup \pgfgdgraphdra...
l.7 \feynmandiagram [horizontal=a to b] {
```
TeXLive was installed through `apt-get` with TikZ-Feynman installed with `tlmgr`.

**Comment by JP-Ellis** [comment]
The issue arises because the version of PGF/TikZ `v3.0.0`, `v3.0.1` and `v3.0.1a` all have a bug which is patched by TikZ-Feynman.  It turns out that the patch is actually incompatible with `v3.0.0` of TikZ and is only compatible with `v3.0.1` and `v3.0.1a`.

I will develop a patch of `v3.0.0` and include it in the next version of TikZ-Feynman.

Until I release the next version, the alternative fix is to upgrade the version of TikZ.  If you installed TeXLive through the net installer, you can upgrade to the latest version of TikZ with:
```sh
tlmgr update pgf
```
and if you installed TeXLive with `apt-get`, then you can get the latest version of TikZ with:
```sh
tlmgr --usermode install pgf
```
or, if it is already installed, you can upgrade it with
```sh
tlmgr --usermode update pgf
```

**Comment by cm-cc** [comment]
thanks, this temporarily fixes the issue in ubuntu 15.10
